source/dotnet/Library/AdaptiveCards/AdaptiveFeatureRegistration.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AdaptiveCards
+{
+    public class AdaptiveFeatureRegistration
+    {
+        public AdaptiveFeatureRegistration()
+        {
+            features[adaptiveCardFeature] = "1.2";
```

Comment:
>= "1.2"; [](start = 42, length = 8)

This again feels like this should be (a) not a hard coded literal (b) configurable

Reply:
not hardcoded makes sense. what do you mean about "configurable"?

---
In reply to: [278321123](https://github.com/Microsoft/AdaptiveCards/pull/2560#discussion_r278321123) [](ancestors = 278321123)

Reply:
Would this need to change beyond 1.2 for eg? WHat happens for 1.3? or is 1.2 used as a min version in this case?

---
In reply to: [278332562](https://github.com/Microsoft/AdaptiveCards/pull/2560#discussion_r278332562) [](ancestors = 278332562,278321123)

Reply:
It needs to be manually updated. It tracks the version of the object model (now tracked in `Globals.cs`)

---
In reply to: [278343475](https://github.com/Microsoft/AdaptiveCards/pull/2560#discussion_r278343475) [](ancestors = 278343475,278332562,278321123)